src/gei/Commands/GenerateScriptCommand.cs

```diff
@@ -38,6 +40,24 @@ public GenerateScriptCommand(OctoLogger log, ISourceGithubApiFactory sourceGithu
                 IsRequired = true,
                 Description = "Uses GH_PAT env variable."
             };
+
+            // GHES migration path
+            var ghesApiUrl = new Option<string>("--ghes-api-url")
+            {
+                IsRequired = false,
+                Description = "If migrating from GHES, the api endpoint for the hostname of your GHES instance. For example: http(s)://api.myghes.com"
```

Comment:
I'd make this "Required if migration from GHES"

Reply:
Updated in https://github.com/github/gh-gei/pull/248/commits/8b741a3a51d6d7a94f5af881c2a3872afa5a17fa